openverse_catalog/dags/providers/provider_workflows.py

```diff
@@ -89,7 +92,8 @@ def __post_init__(self):
     ProviderWorkflow(
         provider_script="finnish_museums",
         start_date=datetime(2020, 9, 1),
-        execution_timeout=timedelta(days=3),
+        pull_timeout=timedelta(days=5),
```

Comment:
FWIW, I tried switching this to `pull_timeout=timedelta(seconds=5)` and running the dag and it did time out after 5 seconds, so that's good. :)